GenericModConfigMenu/Framework/Api.cs

```diff
@@ -467,8 +474,9 @@ private void AddNumericOption<T>(IManifest mod, Func<string> name, Func<string>
         /// <param name="getValue">Get the current value from the mod config.</param>
         /// <param name="setValue">Set a new value in the mod config.</param>
         /// <param name="allowedValues">The values that can be selected, or <c>null</c> to allow any.</param>
+        /// <param name="labels">The values that can be selected, or <c>null</c> to allow any.</param>
         /// <param name="fieldId">The unique field ID used when raising field-changed events, or <c>null</c> to generate a random one.</param>
-        private void AddChoiceOption(IManifest mod, Func<string> name, Func<string> tooltip, Func<string> getValue, Action<string> setValue, string[] allowedValues, string fieldId)
+        private void AddChoiceOption(IManifest mod, Func<string> name, Func<string> tooltip, Func<string> getValue, Action<string> setValue, string[] allowedValues, string[] labels, string fieldId)
```

Comment:
Since the `string[] labels` array is passed in once, any translations won't be updated when the language is changed, unlike the other arguments like `Func<string> name`. I suggest replacing this with a `Func<string, string> formatAllowedValue = null` argument instead.

Reply:
Sounds good. Just updated the PR to support this.